content/blog/2024/09/10/2024-09-10-update-center-brownouts-3.adoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Brownout on Update Center (updates.jenkins.io):\n 11-12 September 2024"
+tags:
+- jenkins
+- jenkins-infra
+- update-center
+authors:
+- smerle33
+opengraph:
+  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
+overview: "We'll be using the new Update Center implementation in production for 1 day"
+discourse: true
+---
+
+== Summary (link:https://en.wikipedia.org/wiki/Wikipedia:Too_long;_didn%27t_read[TL;DR])
+
+Note: this is a follow up of link:https://www.jenkins.io/blog/2024/09/04/update-center-brownouts/[the 6 and 9 September brownouts]
+
+The service https://updates.jenkins.io will switch its implementation to a new system during 1 day:
+
+- Wednesday 11 September 2024 from 02:00pm UTC until Thursday 12 September 2024 02:00pm UTC
+
+All Jenkins users are impacted but should not see any functional change.
+
+⚠️ Please, check that your organization respects the advertised DNS TTL or you might be stuck in the brownout longer that expected.
+
+Under the hood, any HTTP request made to this service will be redirected to a mirror close to user locations during these brownouts.
+
+== What is the "Update Center"?
+
+Jenkins link:https://updates.jenkins.io[Update Center] is a web server at the core of the Jenkins public infrastructure which distributes the plugins, tool installers, and versions index to all Jenkins servers all across the world.
+
+From the installation wizard to regular plugin updates, if you run Jenkins then you use this service under the hood.
+
+Today, it serves around 50 Tb of data (outbound bandwidth) each month from a single virtual machine on AWS, which costs around $6,000 per month.
+
+In order to sustain this service and improve it, the Jenkins infrastructure team has worked relentlessly during the past years to have a new sustainable implementation for this service.
+
+The new Update Center implementation features a highly available system which redirects user requests to a download mirror close to their location.
+Additional information is available in the link:https://github.com/jenkins-infra/helpdesk/issues/2649[GitHub issue].
```

Comment:
imo not really a great example, its a very long issue thread with a description that doesn't look very up to date. Ideally this would have an IEP or some other documentation that is up to date.

Reply:
I concur. But we were outta time to make a proper IEP